app/src/main/java/com/codesoom/assignment/service/TaskServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.codesoom.assignment.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.context.annotation.Configuration;
+
+import com.codesoom.assignment.model.Task;
+
+@Configuration
```

Comment:
`@Configuration`이 용도에 맞지 않게 사용된 것 같습니다. `@Configuration`, `@Service`, `@Component`, `@Controller`에 대해 조사해보시고 차이점을 공부해보세요.
```suggestion
@Service
```

Reply:
제대로 이해하고 사용하지 않았다는 점에서 부끄럽네요... 
공부하는 김에 정리를 해보았습니다
https://velog.io/@sh95119/%EC%8A%A4%ED%94%84%EB%A7%81-%EC%96%B4%EB%85%B8%ED%85%8C%EC%9D%B4%EC%85%98%EC%97%90-%EC%A0%95%EB%A6%AC

Reply:
공식 문서를 읽으려 노력하시는 점에 크게 칭찬해드리고 싶습니다.
`@Configuration`에 대해서는 `@Bean` 공식 문서도 찾아 읽어보세요.